py/torch_tensorrt/fx/converters/acc_ops_converters.py

```diff
@@ -3192,21 +3192,13 @@ def acc_ops_sigmoid(
     kwargs: Dict[str, Argument],
     name: str,
 ) -> Union[TRTTensor, Sequence[TRTTensor]]:
-    input_val = kwargs["input"]
 
-    if not isinstance(input_val, TRTTensor):
-        raise RuntimeError(
-            f"Sigmoid received input {input_val} that is not part "
-            "of the TensorRT region!"
-        )
```

Comment:
It is not removed randomly. Tech checks have been centralized in the core function instead of making the responsibility that of each converter. See the first PR #1867

Reply:
The movement change in #1867 lgtm, left comment on the other pr.